src/handlebars/installation.handlebars

```diff
@@ -62,9 +62,10 @@
             </div>
 
             <div style="margin-top: 3.0em;">
-              <h2 class="bold">Enabling JavaFX / OpenJFX via IcedTea-Web</h2>
-              <p>If you're looking to add Java FX / OpenJFX Support you can currently do so via adding <a href="https://icedtea.classpath.org/wiki/IcedTea-Web">IcedTea-Web</a>.</p>
-              <p>Follow the instructions in this <a href="https://youtu.be/wRRS0LejZuM">IceaTea-Web Installation</a> video.</p>
+              <h2 class="bold">Migration</h2>
+              <p>If you're migrating to an (Adopt)OpenJDK, you can learn about the differences between an Oracle JDK and an OpenJDK in
```

Comment:
I think grammatically (and trademark wise) that `(Adopt)OpenJDK` needs to be `AdoptOpenJDK` and we should drop the `an` in front of AdoptOpenJDK and OpenJDK and Oracle JDK _or_ it needs to read `an XXX JDK binary`.  Hope that all makes sense?